Cockpit 285 (#604)
cockpit-project/cockpit-project.github.io: _posts/2023-02-08-cockpit-285.md

---
title: Cockpit 285
author: garrett
date: '2023-02-08'
tags: cockpit
slug: cockpit-285
category: release

> summary: ''

Author: ```suggestion
summary: Group filtering and creation, cryptographic subpolicies 
```

---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from Cockpit 285:

> ## Accounts: Group improvements

Reviewer: Right, I guess this title was stolen from the general improvments PR.

Users can now create a new group. Existing groups can be filtered by name or GID.

![Screenshot from 2023-02-08 17-59-46]({{ site.baseurl }}/images/285-accounts-group-improvements.png)

Filtering has been added to groups.

![screenshot of group improvements]({{ site.baseurl }}/images/285-accounts-group-improvements-2.png)

## Support a subset of cryptographic policies with subpolicies

> Cockpit now supports setting a limited set of cryptographic policies with subpolicies depending on the RHEL/Fedora version. Subpolicies extend a normal cryptographic policy by for example allowing SHA1 or setting stricter requirements such as OSPP.

Author: ```suggestion
A limited set of cryptographic policies with subpolicies are now in the list of cryptographic choices on Red Hat Enterprise Linux, CentOS Stream, Fedora, and related distributions. Subpolicies extend a standard cryptographic policy by either being more permissive or restrictive than the original policy. Examples include allowing SHA1 or using stricter requirements such as OSPP (Operating System Protection Profile, Common Criteria).
```

![Screenshot from 2023-02-08 09-05-10]({{ site.baseurl }}/images/285-support-a-subset-of-cryptographic-policies-with-subpolicies.png)

## Try it out

Cockpit 285 is available now:

* [For your Linux system](https://cockpit-project.org/running.html)
* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)

* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/285)
* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-b2e6448cf5)

> * [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-4b710b3cf8)

Reviewer: ```suggestion
* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/283)
* [cockpit-machines Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-e1c4b6e2ef)
* [cockpit-machines Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-8f3e0c8d5a)
* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/62)
* [cockpit-podman Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-94d443ecb8)
* [cockpit-podman Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-1c6ec9ef3d)
* [cockpit-ostree Source Tarball](https://github.com/cockpit-project/cockpit-ostree/releases/tag/192)
* [cockpit-ostree Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-9f0b552efb)
* [cockpit-ostree Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-416be6f1bd)
```

We usually also list the rest it seems.

Author: The script usually pulls it in? Not sure why it didn't?

Author: Sure; it works

> * [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)

Author: We need to swap out the links.

> * [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/?releases=F37&packages=cockpit)
> * [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)

Author: ```suggestion
* [Cockpit Fedora 37](https://bodhi.fedoraproject.org/updates/FEDORA-2023-b2e6448cf5)
* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2023-4b710b3cf8)
```
